lib/ok.ex

```diff
@@ -92,65 +74,52 @@ defmodule OK do
   def required(value, _reason), do: {:ok, value}
 
   @doc """
-  Result pipe operator.
-  (Result monad bind operator)
-
-  The result pipe takes the value out of an `{:ok, value}` tuple and passes it as the first argument to the function call on the right.
-
-  ## Examples
-
-      iex> {:ok, 5} ~>> double()
-      {:ok, 10}
-
-      iex> {:error, :previous_bad} ~>> double()
-      {:error, :previous_bad}
-
-      # x is {:ok, 7} defined in `OKTest`.
-      iex> x() ~>> double()
-      {:ok, 14}
-
-  The result pipe is most useful when executing a series of operations that may fail.
-
-      iex> {:ok, 6} ~>> safe_div(3) ~>> double
-      {:ok, 4.0}
-
-      iex> {:ok, 6} ~>> safe_div(0) ~>> double
-      {:error, :zero_division}
-
+  The OK result pipe operator `~>>`, or result monad bind operator, is similar
+  to Elixir's native `|>` except it is used within happy path. It takes the
+  value out of an `{:ok, value}` tuple and passes it as the first argument to
+  the function call on the right.
+  
   It can be used in several ways.
-  Pipe to a local call.
-  This example is the same as calling `double(5)`
+  
+  Pipe to a local call.<br />
+  _(This is equivalent to calling `double(5)`)_
 
-      iex> {:ok, 5} ~>> double
+      iex> {:ok, 5} ~>> double()
       {:ok, 10}
 
-  Pipe to a remote call.
-  This example is the same as calling `OKTest.double(3)`
+  Pipe to a remote call.<br />
+  _(This is equivalent to calling `OKTest.double(3)`)_
 
       iex> {:ok, 5} ~>> OKTest.double()
       {:ok, 10}
 
       iex> {:ok, 5} ~>> __MODULE__.double()
       {:ok, 10}
 
-  Pipe with extra arguments
-  This example is the same as calling `OK.safe_div(3, 4)`
+  Pipe with extra arguments.<br />
+  _(This is equivalent to calling `OK.safe_div(3, 4)`)_
```

Comment:
This is probably a mistake of mine from earlier but I don't see any 3's and 4's should this be equivalent to `OKTest.safe_div(6,2)`